workflows/repeatmasking/RepeatMasking-Workflow-tests.yml

```diff
@@ -0,0 +1,29 @@
+- doc: Test outline for RepeatMasking Workflow
+  job:
+    sequence.fasta:
+      class: File
+      path: https://zenodo.org/record/8116008
```

Comment:
I think it would make more sense to use the Zenodo record used in GTN: https://zenodo.org/record/7085837
And I guess the location should be the direct url to the file, so `https://zenodo.org/api/files/debdbbfd-4739-4f2d-bb79-814ac032c8b5/genome_raw.fasta`
And `path` -> `location` as it's an url now